cuda_bindings/cuda/bindings/cyruntime.pyx.in

```diff
@@ -1885,35 +1894,27 @@ cdef cudaError_t cudaGraphicsVDPAURegisterOutputSurface(cudaGraphicsResource** r
 
 {{if True}}
 
-{{if 'Windows' != platform.system()}}
-cimport cuda.bindings._lib.dlfcn as dlfcn
-{{endif}}
-
 cdef cudaError_t getLocalRuntimeVersion(int* runtimeVersion) except ?cudaErrorCallRequiresNewerDriver nogil:
-    {{if 'Windows' == platform.system()}}
     with gil:
-        raise NotImplementedError('"getLocalRuntimeVersion" is unsupported on Windows')
-    {{else}}
-    # Load
-    handle = dlfcn.dlopen('libcudart.so.13', dlfcn.RTLD_NOW)
-    if handle == NULL:
-        with gil:
-            raise RuntimeError(f'Failed to dlopen libcudart.so.13')
-
-    __cudaRuntimeGetVersion = dlfcn.dlsym(handle, 'cudaRuntimeGetVersion')
+        lib = load_nvidia_dynamic_lib("cudart")
+        filename = Path(lib.abs_path).name
+        handle = <void *><uintptr_t>(lib._handle_uint)
+
+        {{if 'Windows' == platform.system()}}
+        try:
+            __cudaRuntimeGetVersion = <void*><unsigned long long>win32api.GetProcAddress(<uintptr_t>handle, 'cudaRuntimeGetVersion')
+        except:
+            pass
```

Comment:
Does a bare `except:` catch things like a KeyboardInterrupt? If so, should we do something like `except Exception:` or be more specific?